cogs/csgo.py

```diff
@@ -258,19 +257,19 @@ async def pug(self, ctx: commands.Context, *args):
             await player.move_to(channel=team1_channel, reason=f'You are on {team1_captain}\'s Team')
             data = await db.fetch_one('SELECT steam_id FROM users WHERE discord_id = :player',
                                       {"player": str(player.id)})
-            team1_steamIDs.append(data[0])
+            team1_steamIDs[data[0]] = unidecode(player.nick)
```

Comment:
Is the value of dict going to be used somewhere in future?

Reply:
Yep, the steam ID authenticates the user when connecting to the server and then the unidecode(player.nick) is the name that will be forced on the server. Similar to faceit.

Reply:
https://github.com/splewis/get5#match-schema